.github/workflows/canary-integration-test.yml

```diff
@@ -104,26 +104,12 @@ jobs:
     - name: test external script with restricted_auth_permission flag and without having cephfs_filesystem flag
       run: |
         toolbox=$(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
-        # create `radosNamespace1` rados-namespace for `replicapool` rbd data-pool
-        kubectl -n rook-ceph exec $toolbox -- rbd namespace create replicapool/radosNamespace1
-        kubectl -n rook-ceph exec $toolbox -- python3 /etc/ceph/create-external-cluster-resources.py --rbd-data-pool-name replicapool --rados-namespace radosNamespace1 --cluster-name rookStorage --restricted-auth-permission true
-        # test the CRD created rados namespace `namespace-a` using external script
-        kubectl create -f deploy/examples/radosnamespace.yaml
```

Comment:
Why is this gone?

Reply:
As we have removed the per rados-namespace access(1st commit),

because there can be more than one rados namespace in a pool, and we must end up with more than one secret that cannot be accommodated with the StorageClass creation.